### DISCUSSION TITLE
Welcome to FyWorld Discussions!

### DISCUSSION THREAD

**Discussion by Fy-** [Fy-/FyWorld#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Xeros08** [comment]
Just my personal oppinion, but you should put the repository in archival mode if you're putting it on hold.
You can always, and at any time, reverse the action from the Settings of the repo.
